src/MonitorProfiler/Utilities/TypeNameUtilities.cpp

```diff
@@ -8,6 +8,17 @@ TypeNameUtilities::TypeNameUtilities(ICorProfilerInfo12* profilerInfo) : _profil
 {
 }
 
+HRESULT TypeNameUtilities::CacheModuleNames(NameCache& nameCache, ModuleID moduleId)
+{
+    std::shared_ptr<ModuleData> moduleData;
+    if (!nameCache.TryGetModuleData(moduleId, moduleData))
```

Comment:
nit: is this redundant? Looking at the implementation of `TryGetModuleData`, it looks like one of its first checks is the inverse of this (in which case it just returns `S_OK`)